### PR TITLE
Script for running the HLS test suite and updated CI

### PR DESCRIPTION
**Pull request by sjalander**

### CODE REVIEW

File: scripts/run-hls-test.sh [hunk 1]
```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+set -eu
+
+# URL to the benchmark git repository and the commit to be used
+GIT_REPOSITORY=https://github.com/EECS-NTNU/hls-test-suite.git
```
Comment (phate) [inline scripts/run-hls-test.sh:5]: The same issue as for the llvm-test-suite script. Please use: `https://github.com/phate/hls-test-suite.git`

File: scripts/run-hls-test.sh [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+BENCHMARK_RUN_TARGET=run
+
+# We assume that the firtool is in the PATH
+FIRTOOL=firtool
+
+# Check if verilator exists
+if ! command -v verilator &> /dev/null
+then
+	echo "No verilator in ${PATH}" 
+	echo "Consider installing the verilator package for you Linux distro."
```
Comment (phate) [inline scripts/run-hls-test.sh:24]: "for you**r** Linux distro."

File: scripts/run-hls-test.sh [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+done
+
+# Check if firtool exists
+if ! command -v ${FIRTOOL} &> /dev/null
+then
+	echo "${FIRTOOL} is not found."
+	echo "Make sure to use '--firtool COMMAND' to specify which firtool to use."
+	echo "You can use './scripts/build-circt.sh' to build it, if needed."
+	exit 1
+fi
```
Comment (phate) [inline scripts/run-hls-test.sh:75]: Move this check up in the script to the verilator check so we have it all in the same place.

**Review by phate — CHANGES_REQUESTED** [review]

File: scripts/run-hls-test.sh (hunk 2 above)
Comment (sjalander) [reply scripts/run-hls-test.sh]: Fixed

File: scripts/run-hls-test.sh (hunk 3 above)
Comment (sjalander) [reply scripts/run-hls-test.sh:67]: FIRRTOOL is a user parameter, so the test must be below the argument parsing.
I moved the verilator check instead.

**Review by phate — APPROVED** [review]

**Pull request merged**